pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2ClientConfiguration.java

```diff
@@ -431,10 +417,9 @@ public boolean isAuthnRequestSigned() {
 	 *            Web context to transport additional information to the configuration.
 	 */
     protected void init(final String clientName, final WebContext context) {
```

Comment:
@jkacer: I wanted to draw your attention to this change (since 2.0.0-RC1). The `SAML2ClientConfiguration` component is now an `InitializableObject` with its internal `init()` method, which I choose to call from your additional `init(String, WebContext)` one. It may imply a change in your extension.